docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -153,28 +153,6 @@ To install `mbed-cli` bash completion:
 1. Copy the `mbed` script into the  `~/.bash_completion.d` directory (you may need to create this directory first).
 1. Restart the terminal.
 
-### Use Arm Compiler 6 from Mbed Studio
-
-Mbed Studio comes with a free version of Arm Compiler 6 for use with Mbed OS. If you want to use Arm Compiler 6 with Mbed CLI:
-
-1. Install [Mbed Studio](https://os.mbed.com/docs/mbed-studio/latest/introduction/index.html).
-1. Configure the environment variable `ARMC6_PATH`:
-
-    For Mbed Studio 0.5 and later, the Arm Compiler 6 executable is located in the following path:
-
-    - Windows: `C:\Users\<user>\AppData\Local\Mbed Studio\mbed-studio-tools\ac6\bin`
-    - Mac: `/Library/Application Support/Mbed Studio/mbed-studio-tools/ac6/bin`
-
-1. Set the environment variable `ARMLMD_LICENSE_FILE`:
-
-    For Mbed Studio 0.5 and later, the Arm Compiler 6 license file, `ac6-license.dat`, is located in the following path:
-
-    - Windows: `C:\Users\<user>\AppData\Local\Mbed Studio\mbed-studio-tools`
-    - Mac: `/Library/Application Support/Mbed Studio/mbed-studio-tools`
-
-    Note that this environment variable is not managed by Mbed CLI, so you must set it appropriately in your environment.
-
```

Comment:
Any reason we're pulling this out? During the 5.12 release this particular section was explicitly asked for.

Reply:
It's not supported, and will break soon if it hasn't already. Someone (I think Andy P) said it's okay to just pull it out.